consensus/spos/bls/subroundEndRound.go

```diff
@@ -589,20 +589,26 @@ func (sr *subroundEndRound) createAndBroadcastHeaderFinalInfo() {
 }
 
 func (sr *subroundEndRound) createAndBroadcastInvalidSigners(invalidSigners []byte) {
+	leader, errGetLeader := sr.GetLeader()
```

Comment:
maybe also check if it should send the message in the context of a backup machine

Reply:
pushed